sympy/logic/tests/test_boolalg.py

```diff
@@ -505,6 +505,7 @@ def test_to_anf():
             Xor(True, And(Or(x, y), Implies(x, y)), remove_true=False)
     assert to_anf(Nor(x ^ y, x & y), deep=False) == \
             Xor(True, Or(Xor(x, y), And(x, y)), remove_true=False)
+    assert to_anf(x ^ ~(x ^ y ^ ~y)) == False
```

Comment:
You can divide the test as follows,

https://github.com/sympy/sympy/blob/8824e3add70df5dd5ccc130666ba109dc3fe8258/sympy/logic/tests/test_boolalg.py#L553-L560

or it may be helpful to leave the issue number in the comments.